.travis.yml

```diff
@@ -21,70 +21,123 @@ before_install:
 before_script:
   - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | BINARY=golangci-lint sh -s -- -d -b $(go env GOPATH)/bin v1.20.0
   - mkdir -p $HOME/gopath/src/github.com/kubeedge/kubeedge
+  - sudo apt-get install build-essential -y
+  - sudo apt-get install gcc-arm-linux-gnueabi -y
+  - sudo apt-get install upx-ucl -y
+  - sudo apt-get install rsync -y
   - rsync -az ${TRAVIS_BUILD_DIR}/ $HOME/gopath/src/github.com/kubeedge/kubeedge/
   - export TRAVIS_BUILD_DIR=$HOME/gopath/src/github.com/kubeedge/kubeedgee
   - go get github.com/onsi/ginkgo/ginkgo
   - export GO111MODULE=on
   - export GOFLAGS=-mod=vendor
   - cd $HOME/gopath/src/github.com/kubeedge/kubeedge
-  - sudo apt-get install build-essential
-  - sudo apt-get install gcc-arm-linux-gnueabi
-  - sudo apt-get install upx-ucl
 
 install: true
 
 jobs:
   include:
     - stage: "Verify vendor"
+      arch: amd64
       script: make verify
-      name: "check if the vendor is up to date"
+      name: "check if the vendor is up to date on amd64"
+    - script: make verify
+      arch: arm64
+      name: "check if the vendor is up to date on arm64"
 
     - stage: "Test lint"
       script: make cloud_lint
-      name: "cloud_lint"     # names the eighth Tests stage job
+      name: "cloud_lint"
     - script: make bluetoothdevice_lint
       name: "bluetooth_device_lint"
     - script: make keadm_lint
       name: "keadm_lint"
     - script: make edge_lint
       name: "edge_lint"
 
-    - stage: "Test Build"                # naming the Tests stage
-      script: make 
-      name: "builds cloud and edge components"            # names the first Tests stage job
+    - stage: "Test Build"
+      script: make
+      arch: amd64
+      name: "builds cloud and edge components on amd64"
+    - script: make
+      arch: arm64
+      name: "builds cloud and edge components on arm64"
```

Comment:
cross-build should not be included since qemu doesn't support on ARM, cause this script passed I guess cross-build is not included in the default target.